internal/npm_install/npm_install.bzl

```diff
@@ -44,6 +44,29 @@ repository so all files that the package manager depends on must be listed.
         doc = """Environment variables to set before calling the package manager.""",
         default = {},
     ),
+    "generate_local_modules_build_files": attr.bool(
+        default = True,
+        doc = """Enables the BUILD files auto generation for local modules installed with `file:` (npm) or `link:` (yarn)
+
+When using a monorepo it's common to have modules that we want to use locally and
+publish to an external package repository. This can be achieved using a `js_library` rule
+with a `package_name` attribute defined inside the local package `BUILD` file. However,
+if the project relies on the local package dependency with `file:` (npm) or `link:` (yarn) to be used outside Bazel, this
```

Comment:
npm also supports `link:` and yarn also supports `file:`, so i'm not sure why these parentheticals are here?